bin/rpc/rpc_common.mli

```diff
@@ -7,53 +7,35 @@ val active_server_exn : unit -> Dune_rpc.Where.t
 (** Raise an RPC response error. *)
 val raise_rpc_error : Dune_rpc.Response.Error.t -> 'a
 
-(** Make a request and raise an exception if the preparation for the request
-    fails in any way. Returns an [Error] if the response errors. *)
-val request_exn
-  :  Dune_rpc_client.Client.t
-  -> ('a, 'b) Dune_rpc.Decl.request
-  -> 'a
-  -> ('b, Dune_rpc.Response.Error.t) result Fiber.t
-
 (** Cmdliner term for a generic RPC client. *)
 val client_term : Common.Builder.t -> (unit -> 'a Fiber.t) -> 'a
 
 (** Cmdliner argument for a wait flag. *)
 val wait_term : bool Cmdliner.Term.t
 
+type ('a, 'b) message_kind =
+  | Request : ('a, 'b) Dune_rpc.Decl.request -> ('a, 'b) message_kind
```

Comment:
Learning for me: What does `Decl` stand for here? I would guess "Declaration" but the type of the module doesn't obviously look like any sort of declaration. It looks like messages.

Reply:
Decl stands for declaration as you guessed. They are messages in the RPC system, which means here they don't have implementations, just type declarations. The actual implementations of said messages are in `dune_rpc_impl/server.ml`